airflow/config_templates/config.yml

```
@@ -2451,14 +2451,15 @@ scheduler:
     max_tis_per_query:
       description: |
         This changes the batch size of queries in the scheduling main loop.
+        This should not be greater than ``core.parallelism``.
         If this is too high, SQL query performance may be impacted by
         complexity of query predicate, and/or excessive locking.
         Additionally, you may hit the maximum allowable query length for your db.
-        Set this to 0 for no limit (not advised)
+        Setting this to 0 for using the value of ``core.parallelism``
```

This wording is confusing. I think it's trying to say this?:

```suggestion
        Setting this to 0 will use the value of ``core.parallelism``
```

---

I would say _to use_

---

> I would say _to use_

You'd need to change "Setting" --> "Set" if you use that language, which does change the meaning a bit more than intended I think. But both options are better than what was there before :+1: